cluster/src/main/java/com/hartwig/pipeline/Arguments.java

```diff
@@ -12,6 +12,7 @@
     String EMPTY = "";
 
     enum DefaultsProfile {
+        PUBLIC,
```

Comment:
I'm wrestling with whether it would be more obvious if this were called `PLATINUM`.

Reply:
the public profile can be used outside platinum.